### PR TITLE
DOC-3351: Fixed broken links in CE 4.8

### PR DESCRIPTION
**Pull request by taylorharvey-r3**
Made changes up to Configuring responder flows doc.

### CODE REVIEW

File: content/en/platform/corda/4.8/enterprise/cordapps/tutorial-cordapp.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ weight: 20
 ---
 # Run a sample CorDapp
 
-Get started with Corda by running a sample CorDapp. Learn how to download, deploy, launch, interact with, and test a CorDapp before you try [building your own](../../../../../../en/tutorials/corda/4.8/os/template-tutorial/writing-a-cordapp-using-a-template.md), modifying a [template](https://www.corda.net/samples/), or using a [community CorDapp](https://www.corda.net/samples/).
+Get started with Corda by running a sample CorDapp. Learn how to download, deploy, launch, interact with, and test a CorDapp before you try [building your own](../../../../../../en/tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-intro.md), modifying a [template](../../../../../../en/tutorials/corda/4.8/os/template-tutorial/writing-a-cordapp-using-a-template.md), or using a [community CorDapp](https://www.corda.net/samples/).
 
 The local Corda network in the sample includes one notary and two nodes, each representing a party in the network. A Corda node is an individual instance of Corda representing one party in a network. For more information on nodes, see the [node documentation](../../../../../../en/platform/corda/4.8/enterprise/node/component-topology.md/).
```
Comment (taylorharvey-r3) [inline content/en/platform/corda/4.8/enterprise/cordapps/tutorial-cordapp.md:26]:
```suggestion
The local Corda network in the sample includes one notary and two nodes, each representing a party in the network. A Corda node is an individual instance of Corda representing one party in a network. For more information on nodes, see the [node documentation](../../../../../../en/platform/corda/4.8/enterprise/node/component-topology.md).
```

File: content/en/platform/corda/4.8/enterprise/app-upgrade-notes.md [hunk 2]
```diff
@@ -60,25 +60,25 @@ You don't need to perform a manual upgrade for this platform version.
...
 To upgrade your CorDapps to platform version 8, you need to:
 1. [Upgrade existing nodes to version 4.6](#upgrade-existing-nodes-to-version-46).
-2. [Check you're using Corda Gradle plugins version 5.0.12](#check-youre-using-corda-gradle-plugins-version-5012).
+2. [Check that you are using Corda Gradle plugins version 5.0.12](#check-that-you-are-using-corda-gradle-plugins-version-5012).
 
 
 ### Upgrade existing nodes to version 4.6
 
-When upgrading to Corda 4.6 from a previous version, you need to upgrade your nodes because of the operational improvements for [database schema harmonization](release-notes-enterprise.md#database-schema-harmonization) that were introduced as part of this release.
+When upgrading to Corda 4.6 from a previous version, you need to upgrade your nodes because of the operational improvements for [database schema harmonization](../../../../../en/platform/corda/4.8/enterprise/release-notes-enterprise.html#database-schema-harmonization) that were introduced as part of this release.
```
[the remark this answers is not in the document]
Comment (taylorharvey-r3) [reply content/en/platform/corda/4.8/enterprise/app-upgrade-notes.md:68]: Ooh good catch!

[the remark this answers is not in the document]
Comment (taylorharvey-r3) [reply content/en/platform/corda/4.8/enterprise/app-upgrade-notes.md:68]: Looks like this was removed at some point, so will link to the 4.6 release notes.

File: content/en/platform/corda/4.8/enterprise/cordapps/api-contract-constraints.md [hunk 3]
```diff
@@ -118,7 +118,7 @@ If you need to prevent a signer from processing transactions, you can *blacklist
 CorDapps and other attachments installed on a node still run, even if they are signed by a blacklisted key. Only attachments
 received from a peer are affected.
 
-You can also [blacklist keys](../node/setup/corda-configuration-file.md#corda-configuration-file-blacklisted-attachment-signer-keys).
+You can also [blacklist keys](../../../../../../en/platform/corda/4.8/enterprise/node/setup/corda-configuration-file.html#corda-configuration-file-blacklisted-attachment-signer-keys).
```
[the remark this answers is not in the document]
Comment (taylorharvey-r3) [reply content/en/platform/corda/4.8/enterprise/cordapps/api-contract-constraints.md:121]: Will look into this

[the remark this answers is not in the document]
Comment (taylorharvey-r3) [reply content/en/platform/corda/4.8/enterprise/cordapps/api-contract-constraints.md:121]: I'm actually just going to remove this link. There's no heading and barely a mention of blacklisting keys on that page, it doesn't teach you how to do it. This section provides much more info and there's no other documentation for it.

**Review by AnnaK-R3 — APPROVED** [review]

**Pull request merged**